lib/ansible/module_utils/tacp_ansible/tacp_utils.py

```diff
@@ -330,6 +332,48 @@ class FirewallProfileResource(Resource):
     filter_method = "get_firewall_profiles_using_get"
 
 
+class EditApplicationResource(Resource):
```

Comment:
The class name should not be inspired by the resource class name. Please do not follow the cldtx namings as they are at least [insert word here]

I suggest the following: name of the entity + some verb indicating an action + resource

```suggestion
class ApplicationUpdateResource(Resource):
```
Should probably create `UpdateResource` base class in case same code is repeated multiple times.

You can go with whatever you want, but I prefer `Update` rather than `Edit` because it's widely adopted, alings with CRUD and SQL terminology 

Also take this class and place it next to `ApplciationResource` for convenience